cli/js/ops/idna.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+/** https://url.spec.whatwg.org/#idna */
+
+import { sendSync } from "./dispatch_json.ts";
+
+export function domainToAscii(
+  domain: string,
+  { beStrict = false }: { beStrict?: boolean } = {}
+): string {
+  return sendSync("op_domain_to_ascii", { domain, beStrict });
+}
+
+export function domainToUnicode(domain: string): string {
+  return sendSync("op_domain_to_unicode", { domain });
+}
```

Comment:
This is unused right now, but used in the spec for host writing/rendering or something so it would come into play in a later PR. Just setting up a complete IDNA abstraction here so it doesn't have to be revisited.

Reply:
I prefer to have dead code removed - but alternatively you could add a TODO comment.

Reply:
Looking again at the spec, the "host rendering" that this is used for refers to how browsers show hosts in URL bars. Not applicable to us. Removing.